### ISSUE TITLE
How to track progress of Circle?

### ISSUE THREAD

**Issue by mistrydarshan99** [faruktoptas/kooldown#2]
Library looks awesome but before I have used this library I have one question.

Question: 
1) Is there any scope to track progress of circle?
example : If animation of circle complete 10% then I will get any int or long value how much circle is cover through animation.

**Comment by faruktoptas** [comment]
Thanks :) Do you want to get the progress value from the view at a certain time or via a listener continuously?

**Comment by mistrydarshan99** [comment]
Yes it would be good if you write some logic in **drawCircle()** function to emit progress value.

**Comment by faruktoptas** [comment]
it should not be triggered each time drawCircle method called. It may cause performance problems. There must be configurable minimum(default value may be 100ms) timeout to trigger this method.
If you want to work on this issue PR's welcome :)
